packages/serde/serde.0.0.1/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "A serialization framework for OCaml"
+description:
+  "Serde is a serialization framework for OCaml that runs on the principle of maximum efficiency and user ergonomics while maintaining format flexibility. Internally Serde uses an intermediate language that can be cheaply mapped to and from different target formats like JSON, XML, or S-expressions, which makes Serde good for transcoding data as well."
+maintainer: ["Leandro Ostera <leandro@abstractmachines.dev>"]
+authors: ["Leandro Ostera <leandro@abstractmachines.dev>"]
+license: "MIT"
+homepage: "https://github.com/leostera/serde.ml"
+bug-reports: "https://github.com/leostera/serde.ml/issues"
+depends: [
+  "ocaml" {>= "4.12.0"}
+  "dune" {>= "3.5" & >= "3.5"}
+  "ocamlformat" {>= "0.17.0"}
+  "ppx_inline_test" {>= "0.16.0"}
```

Comment:
I'm not seeing an use of those two packages. Where are they used?
```suggestion
```

Reply:
I suspect `ocamlformat` should be marked as `dev`?